wiki/People/Beatmap_Nomination_Group/Beatmap_Nominator_Rules/en.md

```diff
@@ -1,14 +1,14 @@
 # Beatmap Nominator Rules
 
-The rules listed here affect what Beatmap Nominators can and cannot do when nominating beatmaps for the Ranked status as well as set the tone for the general conduct expected from them. 
+The rules listed here affect what Beatmap Nominators can and cannot do when nominating beatmaps as well as set the tone for the general conduct expected from them. 
 
-These rules are the result of discussion within the [Quality Assurance Team](https://osu.ppy.sh/g/7) (QAT) and have taken into account feedback from the [Beatmap Nomination Group](https://osu.ppy.sh/g/28) (BNG) upon their proposal.
+These rules are the result of discussion within the [Quality Assurance Team](https://osu.ppy.sh/g/7) and have taken into account feedback from the [Beatmap Nominators](https://osu.ppy.sh/g/28) upon their proposal.
 
-As part of the BNG, Beatmap Nominators possess the tools to nominate beatmaps and shape the world of osu! by promoting new content towards being included in the official listings, for this reason Beatmap Nominators have to abide by the following rules at all times:
+Beatmap Nominators possess the tools to nominate beatmaps and shape the world of osu! by promoting new content towards being included in the official listings, for this reason they have to abide by the following rules at all times:
 
 ## Rules
 
-Rules are exactly that: rules. Violating any of these may get the respective Beatmap Nominator expelled permanently from the Beatmap Nomination Group.
+Rules are exactly that: rules. Violating any of these may get the respective Beatmap Nominator expelled permanently from the Beatmap Nominators.
 
 Violations of these rules may be reported in [this thread](https://osu.ppy.sh/forum/t/447428) or to a member of the [Quality Assurance Team](https://osu.ppy.sh/g/7) directly.
```

Comment:
Refer to new forum please. (https://osu.ppy.sh/community/forums/topics/447428) Also I wonder if this is even the right place to report nominator rule violations